### ISSUE TITLE
Create a project-agnostic DCA demo

### ISSUE THREAD

**Issue by afwillia** [Sage-Bionetworks/data_curator#364]
**Is your feature request related to a problem? Please describe.**
- [ ] Create a project-agnostic demo instance for production version of DCA. Create the oauth client with IT.

**How important is this feature?** Select from the options below:
• 🌗 Medium - can do work without it; but it's important (e.g. to save time or for convenience)

**When will use cases depending on this become relevant?** Select from the options below:
• Short-term - 2-4 weeks

**Comment by ychae** [comment]
AC: app that you can access on shinyapps.io

**Comment by rrchai** [comment]
@milen-sage @afwillia 
The current oauth client of this repo (`data_curator`) has the demo instance url configured, which is `https://sagebio.shinyapps.io/data_curator-demo`. We should not need to request another demo instance url for now, unless we need more. 
FYI: This oauth client also accepts other two urls: `data_curator-testing` and `data_curator-testing2`.

One thing needed the demo instance is a synapse project/fileview that contains 1-2 projects from multiple data centers for demoing, HTAN, CSBC, etc. Currently, the demo instance is using HTAN configuration.

**Comment by milen-sage** [comment]
@rrchai just confirming, we are not using these demo instances for features in development, and we can setup one of these urls with an app deployed with latest stable production code (both DCA and schematic); and make it project agnostic (e.g. as you indicated with an example fileview containing a couple of projects, datasets in each, etc.)?

**Comment by afwillia** [comment]
Here is a mock fileview we can use for orientation. It currently has one project with a biospecimen template
https://www.synapse.org/#!Synapse:syn33715412/tables/

**Comment by rrchai** [comment]
Options:

1. Use the upstream data curator repo to deploy the demo instance:

   - [ ] Replace the mock-up schematic config (`schematic_config.yml`) by using demo (project-agnostic) configurations (fileview, data model etc.
   - [ ] Decide the instance site: https://sagebio.shinyapps.io/data_curator-demo` or add new url for the new demo instance (need to modify the client and re-verify)
   - [ ] Update the workflow to deploy the demo instance by release tag, same as `prod`

2. Use new repo, i.e `data_curator_demo`  to deploy the demo instance:
   - [ ] Fork the upstream `data_curator` repo
   - [ ] Create a new client with demo instance url, i.e `https://sagebio.shinyapps.io/data_curator_demo` (I will discard the `https://sagebio.shinyapps.io/data_curator-demo` url once dashboard feature is merged.
   - [ ] Update schematic config (`schematic_config.yml`) by using demo (project-agnostic) configurations (fileview, data model etc.
   - [ ] Update the workflow to deploy the demo instance by release tags, just like other forks.

**Comment by afwillia** [comment]
The [demo instance](https://sagebio.shinyapps.io/data_curator-testing2/) is up and running using the [example data model](https://github.com/Sage-Bionetworks/data-models) and a [mock fileview](https://www.synapse.org/#!Synapse:syn33715412/tables/). It's based on a new [demo-instance](https://github.com/Sage-Bionetworks/data_curator/tree/demo-instance) branch in the main DCA repo.

**Comment by milen-sage** [comment]
Great!

@afwillia could you add the [FAIR Data synapse team](https://www.synapse.org/#!Team:3438869) as an admin to the demo [instance project](https://www.synapse.org/#!Synapse:syn33715357/files/)?

That way we can add users with the right permissions to the project and fileview; otherwise they'd see "Generating" in drop downs.

<img width="1338" alt="Screen Shot 2022-09-09 at 1 59 38 PM" src="https://user-images.githubusercontent.com/41303818/189444851-edda185c-1d53-45c2-b51c-cf0b00e6f8f6.png">

And could you remove the HTAN logo from the top?

**Comment by rrchai** [comment]
@afwillia and @milen-sage 
Instead of keeping another life-long branch and different instances, how about making this repo as the project-agonistic data curator? I think it's the purpose of this repo and will be easier to maintain. Here are some minor remaining steps to achieve this:
- [ ] Update the `schematic_config.yml` to use demo project
- [ ] No need to use another instance or update workflow anymore to deploy for demo site
- [ ] Remove the HTAN logo
- [ ] create the pr against develop and merge into develop
- [ ] then we merge into main

So the production of this repo will become demo app.
